packages/features/ee/workflows/components/TextEditor/stylesEditor.css

```diff
@@ -0,0 +1,430 @@
+
+.editor a {
```

Comment:
css classes mainly come from here: https://codesandbox.io/s/lexical-rich-text-example-5tncvy?file=/src/styles.css but adapted them and removed everything that was not needed